python/docs/source/index.rst

```diff
@@ -10,6 +10,10 @@ transmit solutions, observations, status and debugging messages, as
 well as receive messages from the host operating system, such as
 differential corrections and the almanac.
 
+Please see [the message
```

Comment:
Is this correct syntax for a hyperlink in RST?
http://sphinx-doc.org/rest.html#hyperlinks says not, but I guess what matters is how it comes out after compilation.